iati/resources.py

```diff
@@ -396,8 +475,20 @@ def load_as_string(path):
     Todo:
         Pass in PACKAGE as a default parameter, so that this code can be used by other library modules (e.g. iati.fetch).
 
+        Add test to load a dataset saved in non-UTF-8 formats. This should include `UTF-16LE`, `UTF-16BE` and `windows-1252` since there are published Datasets using these encodings.
```

Comment:
Why hasn't this test been done?

Reply:
57536c9793d6891978be81eca682ae3312b0da8c